docs/application/dotnet/guides/nui/nui-components/Pagination.md

```diff
@@ -1,161 +1,262 @@
 # Pagination
 
-Pagination is a common component supporting multiple pages. It shows the number of pages available and the currently active page.
+
+Pagination is a component for making navigation through multipage windows more user-friendly.
+It can be used to display the number of available pages and marking the currently viewed one.
+The figure below illustrates the use of a simple pagination - it shows 5 available pages, of which the first one is currently displayed.
 
 ![Pagination](./media/Pagination.png)
 
-## Create with property
+Pagination can be customized for size and the background color as well as for using images or solid colors for indicators.
+Navigation within the pages occurs by attaching to it the window key event as described at the end of this section.
 
-To create a Pagination using property, follow these steps:
 
-1. Create Pagination using the default constructor:
+In the following sections on this guide one can find:
+- [the main pagination properties](#paginationProperties)
+- how to create a pagination
+   - [using properties](#createWithProperty)
+   - [using style](#createWithStyle)
+   - [using a custom style](#createWithCustomStyle)
+- [how to connect pagination with a KeyEvent](#connectWithKeyEvent)
 
-    ```cs
-    pagination = new Pagination();
-    ```
 
-2. Set the Pagination property:
+<a name="paginationProperties"></a>
+## Pagination properties
+
+The following table lists the properties specific to the Pagination class:
+
+**Table: Pagination properties**
+
+| Property                  | Type               | Description                                   |
+|---------------------------|--------------------|-----------------------------------------------|
+| `IndicatorSize`           | `Size`             | The absolute size of the indicator.           |
+| `IndicatorSpacing`        | `int`              | The space between the indicators (in pixels). |              
+| `IndicatorCount`          | `int`              | The count of the indicators.                  |              
+| `IndicatorImageUrl`       | `Selector<string>` | The background resource of the indicator.     |              
+| `IndicatorColor`          | `Color`            | The color of the indicator.                   |              
+| `SelectedIndicatorColor`  | `Color`            | The color of the selected indicator.          |              
+| `SelectedIndex`           | `int`              | The index of the select indicator.            |              
+
+[View](https://docs.tizen.org/application/dotnet/guides/nui/view/) is the (indirect) base class of pagination, and you can also
```

Comment:
Global: Please use relative paths: 

https://github.com/Samsung/tizen-docs/pull/1366#discussion_r534619626

```suggestion

[View](https://docs.tizen.org/application/dotnet/guides/nui/view/) is the indirect base class of pagination, and you can also use its properties as shown in the following example:

Reply:
I removed the ()
are you ok with the sentence like that:
[Tizen.NUI.BaseComponents.View](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html) is the indirect base class of pagination, and you can also use its properties as shown in the examples in this section.

there is no 'the following example', that's why I changed it

Reply:
[Tizen.NUI.BaseComponents.View](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html) is the indirect base class of pagination, and you can also use its properties as shown in the examples:

Reply:
I would end the sentence with a dot because it is followed by a new section, not an example, but I changed it as you wanted.